restapi.go

```diff
@@ -2906,15 +2906,7 @@ func (s *Session) StageInstance(channelID string) (si *StageInstance, err error)
 // channelID : The ID of the Stage channel
 // topic : The new topic of the Stage instance (1-120 characters). Set empty to keep the current topic.
 // privacyLevel : The new privacy level of the Stage instance. Set nil to keep the current privacy level.
-func (s *Session) StageInstanceEdit(channelID string, instance *StageInstance) (si *StageInstance, err error) {
-	data := struct {
-		Topic string `json:"topic,omitempty"`
-		// PrivacyLevel of the Stage instance (default GUILD_ONLY)
-		PrivacyLevel *StageInstancePrivacyLevel `json:"privacy_level,omitempty"`
-	}{
-		Topic:        instance.Topic,
-		PrivacyLevel: &instance.PrivacyLevel,
-	}
+func (s *Session) StageInstanceEdit(channelID string, data *StageInstanceParams) (si *StageInstance, err error) {
```

Comment:
```suggestion
func (s *Session) StageInstanceEdit(channelID string, data StageInstanceParams) (si *StageInstance, err error) {
```

Reply:
We always use a pointer on all library for making a request (refer to others `restapi.go`)